### ISSUE TITLE
Problem with oled-exp cursor command

### ISSUE THREAD

**Issue by BeneKenobi** [OnionIoT/i2c-exp-driver#2]
I came across a problem with the `oled-exp` cursor command:
When I run the following expression (via ssh):

```
oled-exp cursor 2,5 write "test"
```

I get the following (expected) output:

```
> Setting cursor to (2, 5)
> Writing 'test' to display
```

but on the display, the text is displayed at position 2,0.

This happens on every use of the cursor command for me, not only on row 2. Looks like setting the column isn't working at all.

**Comment by greenbreakfast** [comment]
Fixed in commit 2db5f36011e09b66c9ec7ea3aca33756c4b90ad6
Will push into a firmware build soon

**Comment by greenbreakfast** [comment]
Fixed in Omega firmware b236
